build.gradle

```diff
@@ -112,7 +112,6 @@ dependencies {
     implementation 'com.squareup.okhttp3:logging-interceptor:4.9.3'
     implementation 'com.google.code.gson:gson:2.9.0'
     implementation 'io.gsonfire:gson-fire:1.8.5'
-    implementation 'javax.ws.rs:jsr311-api:1.1.1'
```

Comment:
I'm not sure why both jsr311-api and rs-api were added by the openapi generator - this was causing conflicting interface declarations within downstream consumers.
```
jsr311-api - This is the official spec jar for the JAX-RS 1.x line.
javax.ws.rs-api - This is the official spec jar for the JAX-RS 2.x line.
```
The project does not depend upon the 1.x spec, so this dependency was removed and still compiles / functions fine.